hub/templates/hub/area/_opinion_data.html

```diff
@@ -0,0 +1,14 @@
+{% for item in category.data %}
```

Comment:
If we really must have a loop, I’d be tempted to move it out into `area.html`, so that this file works just the same as `_place_data.html`, ie: given a single data item (with a label, value, average, source, etc), it prints it out as a card.